Homework/src/by/tms/Lesson_6/Computer.java

```diff
@@ -0,0 +1,125 @@
+package by.tms.Lesson_6;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Computer {
+
+    private String processor;
+    private String operatingSystem;
+    private String hardDisk;
+    private int resourceOfWork;
+    private boolean isBurnt;
+    private boolean isWork;
+
+    private final int maxResources = 4;
+
+    public Computer(String processor, String operatingSystem, String hardDisk) {
+        this.processor = processor;
+        this.operatingSystem = operatingSystem;
+        this.hardDisk = hardDisk;
+        resourceOfWork = 0;
+        isBurnt = false;
+        isWork = false;
+    }
+
+    public void printInformation() {
+        System.out.println("Processor: " + processor + ", operating system: " + operatingSystem + ", hard disk: " +
+                hardDisk + ", resources: " + resourceOfWork);
+    }
+
+    public boolean randomNumber(Scanner scanner) {
```

Comment:
name methods properly please. randomNumber should return some number ) but here it returns boolean :) unexpectedly )))